applications/BlockAIDS.md

```diff
@@ -0,0 +1,297 @@
+
+Ruslan Poverga - Co-Founder and CEO of NGO Positive Initiative
+https://www.facebook.com/poverga
+
+
+
+## Development Status :open_book:
+
+Our team has developed an electronic platform for people living with HIV from Moldova, as a pilot country (web2 version). Our electronic software counts 50k active users and over 1 mln transactions (medical records). You may find below the link to our PoC:
+https://app.powerbi.com/view?r=eyJrIjoiZDc4MDk2ZTctYTRmNy00MTdlLWIxNDEtNmQ3ZjE4MWZjNWUxIiwidCI6IjEzY2ZkMmU4LWFhZTktNDk5OS1hZDk0LWFhZmQxMmVjMmRmNiIsImMiOjh9
+
+We have also developed an NFT Platform dedicated to people living with HIV (a crowdfunding tool):
+https://nft.blockaids.world
+
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 3 months
+- **Full-Time Equivalent (FTE):**  4 FTE
+- **Total Costs:** $30000
+
+### Milestone 1 Fungible Token Smart Contract (FT minter) and wallet authorization system
+
+- **Estimated duration:** 1.5 month
+- **FTE:**  3
+- **Costs:** 15,000 USD
+
+> :exclamation: **The default deliverables 0a-0d below are mandatory for all milestones**, and deliverable 0e at least for the last one. 
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| **0a.** | License  MIT  |
+| **0b.** | Documentation | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how a user can login and use Fungible tokens |
+| **0c.** | Testing and Testing Guide | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker | We will provide Dockerfile and docker image to run the website in one command line, and you can to test those modules |
+| 1. | User Management module development | We will provide login function, so users can login to our web app, using polkadot account. Blockchain Framework: Substrate / Polkadot / Kusama. Development Language: Rust/Ink! . User interface development: React.JS. Backend: NestJS |
+| 2. | Fungible token smart contract | We will create Fungible token token smart contract in order to be able to reward users for their activities on our web app. Blockchain Framework: Substrate / Polkadot / Kusama. Development Language: Rust/Ink! |
+| 3. | Fungible token token balance display functionality | Users logged in with their wallet will be able to see Fungible token balance. User interface development: React.JS. Backend: NestJS. Development Language: Rust/Ink! |
+| 4. | Tansfer module | Trigger a payment by initiating a transfer, that will transfer Fungible token to other user's wallet. User interface development: React.JS. Backend: NestJS. Development Language: Rust/Ink!  | 
+| 5. | Withdraw module | Withdraw the Fungible token from our web app to user's own wallet. User interface development: React.JS. Backend: NestJS. Development Language: Rust/Ink! | 
```

Comment:
As you in the title already indicate, this milestone is basically about implementing (or probably rather re-using) an NFT pallet (and implementing a matching frontend), such as the [NFT FRAME pallet](https://polkadot.js.org/docs/substrate/extrinsics/#nfts) without adding anything novel to it. IMO it doesn't bring a lot of additional value to the ecosystem.